Content.Server/Singularity/EntitySystems/EventHorizonSystem.cs

```diff
@@ -402,6 +412,17 @@ public static void PreventBreach<TComp>(EntityUid uid, TComp comp, ref EventHori
             PreventConsume(uid, comp, ref args);
     }
 
+    // ES Start
+    public void PreventConsumeAlive(Entity<MobStateComponent> ent, ref EventHorizonAttemptConsumeEntityEvent args)
+    {
+        if (args.Cancelled)
+            return;
+        if (!MobStateSystem.IsDead(ent.Owner, ent.Comp) && args.EventHorizon.consumeAliveMobs)
+            PreventConsume(ent.Owner, ent.Comp, ref args);
+    }
```

Comment:
i think this is confusingly written (hence the error) when it should look more like
```suggestion
        if (args.Cancelled || args.EventHorizon.consumeAliveMobs)
            return;
        if (MobStateSystem.IsDead(ent.Owner, ent.Comp))
            return;
        PreventConsume(ent.Owner, ent.Comp, ref args);
    }
```